client/app/scripts/superdesk-archive/directives.js

```diff
@@ -396,6 +396,21 @@
                 }
             };
         }])
+        .directive('sdDraggableItem', function() {
+            return {
+                link: function(scope, elem) {
+                    if (scope.item) {
```

Comment:
Ah, if that's the case then it's probably not needed, since it's clear what `item` is in the template.